src/ontology/mondo-edit.obo

```diff
@@ -255148,25 +255148,17 @@ synonym: "epilepsy, juvenile absence, susceptibility to, 1" RELATED [MONDO:Lexic
 synonym: "epilepsy, juvenile absence, susceptibility to, type 1" EXACT [MONDORULE:1, OMIM:607631]
 synonym: "JAE" EXACT ABBREVIATION [Orphanet:1941]
 synonym: "susceptibility to juvenile absence epilepsy 1" RELATED [OMIM:607631]
-xref: DOID:0060172 {source="MONDO:equivalentTo"}
 xref: GARD:2162 {source="Orphanet:1941"}
 xref: ICD10CM:G40.3 {source="Orphanet:1941", source="Orphanet:1941/attributed", source="Orphanet:1941/ntbt"}
 xref: MESH:C535495 {source="Orphanet:1941", source="Orphanet:1941/e"}
-xref: NCIT:C129868 {source="MONDO:equivalentTo"}
-xref: OMIMPS:607631 {source="MONDO:equivalentTo"}
-xref: Orphanet:1941 {source="MONDO:equivalentTo", source="OMIM:607631"}
-xref: SCTID:230413002 {source="MONDO:equivalentTo"}
+xref: OMIMPS:607631 {source="MONDO:obsoleteEquivalent"}
```

Comment:
I think this OMIMPS should exist in Mondo and group the susceptibility classes: https://www.omim.org/phenotypicSeries/PS607631

In cases like this, I have created a new class that is equiv to the OMIMPS

Reply:
@nicolevasilevsky do you mean create a new Mondo class and `xref: OMIMPS:607631 {source="MONDO:obsoleteEquivalent"}` to the new class?

Reply:
Create a new class that is named 'epilepsy, juvenile absence, susceptibility to' that is equivalent to OMIMPS:607631 (ie `xref: OMIMPS:607631 {source="MONDO:equivalentTo"} and that groups the children terms in this PS